tests/coverage/run/i18233/i18233.scoverage.check

```diff
@@ -42,23 +42,6 @@ Foo$
 Object
 <empty>.Foo$
 render
-54
-58
-4
-List
-Ident
-false
-0
-false
-List
```

Comment:
Not sure if these changes in the coverage are fine. This seems to relate with the `List` accessor in `Predef`. Coverage might not be tracking it anymore because now it is a `val` instead of a `def` and there fore it does not consider it as a method call.